doc/conf.py

```diff
@@ -15,6 +15,10 @@
 
 import pkg_resources
 from datetime import datetime
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '/doc/'))
```

Comment:
why do we need this?

Reply:
`sphinx` has issues importing modules from the file system. You have to explicitly define where it should look https://github.com/sphinx-doc/sphinx/issues/2390